modules/ROOT/pages/_partials/eol-warning.adoc

```diff
@@ -2,9 +2,9 @@
 [WARNING]
 ====
 Standard Support for Mule 4.1 ended on November 2, 2020, and
```

Comment:
Note that we are keeping this long-winded warning in v4.1 and v4.2 but very likely will not use it for v4.3 and above.

This notice was out of date. The fix below clarifies what milestones have been reached already.